liquidhaskell.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      2.4
 name:               liquidhaskell
-version:            0.8.10.7.1
+version:            0.8.10.7.1.1
```

Comment:
A version chosen not to interfere with any current versions...

Reply:
```suggestion
version:            0.8.10.7.1
```